weatherreport.cpp

```diff
@@ -36,10 +32,29 @@ namespace WeatherSpace
             return 52;
         }
     };
+
+    // New stub added to expose the bug (high precip + high wind)
+    class StormyStub : public IWeatherSensor {
```

Comment:
This stub has an intuitive name, indicating what kind of weather it stubs.
Rename the original `SensorStub` also in this way